src/reputation/related_models/bounty.py

```diff
@@ -36,6 +36,14 @@ class Bounty(DefaultModel):
         (CANCELLED, CANCELLED),
         (EXPIRED, EXPIRED),
     )
+    REVIEW_TYPE = "REVIEW"
+    ANSWER_TYPE = "ANSWER"
+    OTHER_TYPE = "GENERIC_COMMENT"
+    type_choices = (
+        (REVIEW_TYPE, REVIEW_TYPE),
+        (ANSWER_TYPE, ANSWER_TYPE),
+        (OTHER_TYPE, OTHER_TYPE),
+    )
```

Comment:
Nit: Should we use `TextChoice` here?

Example: https://github.com/ResearchHub/researchhub-backend/blob/fde9c45981bc966d6f6278ce21de4a7f6022e162/src/user/related_models/user_verification_model.py#L14

Reply:
Made change